src/app.html

```diff
@@ -38,15 +40,21 @@
 						document.documentElement.classList.add(e);
 					});
 				} else if (localStorage.theme && localStorage.theme === 'system') {
-					systemTheme = window.matchMedia('(prefers-color-scheme: dark)').matches;
-					document.documentElement.classList.add(systemTheme ? 'dark' : 'light');
+					document.documentElement.classList.add(isSystemDark ? 'dark' : 'light');
 				} else if (localStorage.theme && localStorage.theme === 'her') {
 					document.documentElement.classList.add('dark');
 					document.documentElement.classList.add('her');
 				} else {
 					document.documentElement.classList.add('dark');
 				}
 
+				if (
+					localStorage?.theme.includes('dark') ||
+					(localStorage.theme === 'system' && isSystemDark)
+				) {
+					window.themeColor.content = '#0d0d0d';
```

Comment:
Shouldn't this also be added to the listener?

Reply:
I've added it to the listener!